Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM sonarsource/sonar-scanner-cli:5.0
+FROM sonarsource/sonar-scanner-cli:5.0.0
```

Comment:
```suggestion
FROM sonarsource/sonar-scanner-cli:5.0

USER root
```

Reply:
Teh versions apparently published are:

![image](https://github.com/SonarSource/sonarcloud-github-action/assets/84329129/4c4d9d53-ebc9-4407-97aa-d7e2772bd1e4)

So I think this PR not have much sense